src/frontend/Komponenter/Verktøy/BeregningsskjemaSide/typer.ts

```diff
@@ -0,0 +1,32 @@
+export enum TiProsentAvvik {
+    INGEN_VERDI = '',
+    OPP = '10% avvik opp',
+    NED = '10% avvik ned',
+    NEI = 'Ikke 10% avvik',
+    UNDER_HALV_G = 'Under halv G',
+}
+
+export interface Beregning {
+    periode: {
+        årstall: string;
+        måned: string;
+    };
+    arbeidsgivere: { navn: string; verdi: number }[];
+    årslønn: number;
+    redusertEtter: number;
+    avvik: TiProsentAvvik;
+    beregnetfra: boolean;
+}
+
+export interface Periode {
```

Comment:
```suggestion
export interface InntektPeriode {
```
Stemmer det at dette er en inntektperiode? I såfall kan det være nice å få frem dette i navnet på interfacet